packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,165 @@
+| Issuer        | The Oauth2 URL for the Okta Org, such as `https://oie-123456.oktapreview.com/oauth2/default`. |
+| Client ID     | The ID of the Okta Application Integration from the Okta Admin console.  |
+| Client secret | An optional shared secret for accessing the Application Integration. |
+| Redirect URI  | A callback URI for launching the mobile app, such as `com.example.oie-123456:/callback`. |
+| Scopes        | A space separated list of the requested access scopes for the app. |
+
+The configuration information in a shipping app is usually static. You can initialize the
+configuration values directly in the code or by reading them in from a file. During
+development you may want to provide a way to edit configuration values.
+
+<StackSnippet snippet="loadingaconfiguration" />
+
+### Start sign-in
+
+Start the sign-in flow by first creating an SDK client using a configuration, and then requesting the first step.
+
+<StackSnippet snippet="initializingsdksession" />
+
+### Process the response
+
+The steps for processing a response are:
+1. Check for a successful sign-in.
+1. Check for messages, such as an invalid password.
+1. Check for remediations.
+1. Process the remediations.
+1. Process the authenticators.
+
+After the user enters the required information, update the remediation and request the next step.
+
+<StackSnippet snippet="processresponse" />
+
+### Complete sign in
+
+Check the response for a successful sign-in at each step of the flow. When the user is
+signed-in, exchange the session token for an access token, and then exit the flow.
+
+<StackSnippet snippet="gettingatoken" />
+
+### Sign out a user
+
+Sign out a user by revoking the access token granted by signing in, and by revoking a
+refresh token if one exists.
+
+<StackSnippet snippet="signingout" />
+
+## Sign-in flow UI
+
+There are two general approaches for adding the sign-in flow to your app: add a fixed
+number of authentication methods possibly in a fixed order, or generate the user interface
+dynamically based on the response at each step.
+
+For a fixed number of methods, you create appropriate views and populate them from the
+response. Using fixed methods does introduce risk as authentication methods may be added
+or removed by the Okta administrator. There are three requirements to reduce that risk:
+
+- The code in the app can be updated before policy changes are enabled.
+- The updated app can be distributed to users before the policy changes are enabled.
+- You can ensure that the app is updated on all your users' devices.
+
+The usual way to satisfy the last condition is with an enterprise managed workforce app.
+Also note that there's additional risk as external events may require an immediate policy
+change that can result in the inability of mobile users to sign-in.
+
+For consumer apps, and to reduce the risk for enterprise apps, a safer choice is building
+the user interface from the current sign-in step, by presenting a dynamic UI.
```

Comment:
We should emphasize this is the better approach! And list benefits.

Reply:
That happens in a [later phase of the project](https://docs.google.com/document/d/11F8u2uxmiFoI9-pcvJYdglD7HVRUmuD927C7qeUW5rQ/edit#heading=h.erkvwiam67vk). This phase is about creating the background information for the SDKs and the argument flow for "do it dynamically". But before we can push dynamic UI we need a way for them to implement it easily (Phase 3).